app/packs/src/stores/alt/actions/ElementActions.js

```diff
@@ -485,6 +485,17 @@ class ElementActions {
     return cellLineSample;
   }
 
+  copyCellLineFromId(id,collectionId ) {
+    return (dispatch) => {
+      CellLinesFetcher.copyCellLine(id,collectionId)
+        .then((result) => {
+          dispatch(result);
+        }).catch((errorMessage) => {
+          console.log(errorMessage);
```

Comment:
I think we should add a response to the user that something went wrong

Reply:
Is this an error that can occur normally or should it not actually fail? In the latter case, I think that logging is enough. Especially since all other operations in the `ElementAction` only log to the console in the case of an error. What do you think?